[BUG] Create azure profile

**Describe the bug**
I am trying to create an Azure profile, but get the following error: `panic: unable to redefine 'e' shorthand in "azure" flagset: it's already used for "storage-env" flag`

**To Reproduce**
1. Type the command `kanctl create profile azure`
2. Press [enter]
3. The following error appears: 

```
$ kanctl create profile azure
panic: unable to redefine 'e' shorthand in "azure" flagset: it's already used for "storage-env" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc000465300, 0xc0006d9220)
        /root/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:874 +0x3f5
github.com/spf13/pflag.(*FlagSet).AddFlagSet.func1(0xc0006d9220)
        /root/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:887 +0x65
github.com/spf13/pflag.(*FlagSet).VisitAll(0x5, 0xc0006bfce0)
        /root/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:290 +0xe3
github.com/spf13/pflag.(*FlagSet).AddFlagSet(0xc0006e6780, 0x5)
        /root/go/pkg/mod/github.com/spf13/pflag@v1.0.5/flag.go:885 +0x3d
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc0006e6780)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:1658 +0x65
github.com/spf13/cobra.(*Command).InitDefaultHelpFlag(0xc0004ee4c0)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:1032 +0x25
github.com/spf13/cobra.(*Command).execute(0xc0006e6780, {0x48434f0, 0x0, 0x0})
        /root/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:780 +0x107
github.com/spf13/cobra.(*Command).ExecuteC(0xc00061f400)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
        /root/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/kanisterio/kanister/pkg/kanctl.Execute()
        /codefresh/volume/kanister/pkg/kanctl/kanctl.go:46 +0x25
main.main()
        /codefresh/volume/kanister/cmd/kanctl/main.go:28 +0x17
```
**Expected behavior**
The user gets information about how to create an Azure profile (instead of s3 or gcp).

**Environment**
Kubernetes Version/Provider: v1.22.5
Storage Provider: Azure
Cluster Size (#nodes): 5
Data Size: A few GBs

--- a/pkg/kanctl/profile.go
+++ b/pkg/kanctl/profile.go
@@ -140,7 +140,7 @@ func newAzureProfileCmd() *cobra.Command {
 
 	cmd.Flags().StringP(AzureStorageAccountFlag, "a", "", "Storage account name of the azure storage")
 	cmd.Flags().StringP(AzureStorageKeyFlag, "s", "", "Storage account key of the azure storage")
-	cmd.Flags().StringP(AzureStorageEnvFlag, "e", "", "The Azure cloud environment")
+	cmd.Flags().String(AzureStorageEnvFlag, "", "The Azure cloud environment")
 
 	_ = cmd.MarkFlagRequired(AzureStorageAccountFlag)
 	_ = cmd.MarkFlagRequired(AzureStorageKeyFlag)

